test/foundry/fork/arbitrum/ArbitrumSmartAccountUpgradeTest.t.sol

```diff
@@ -61,95 +126,32 @@ contract Test_SmartAccountV2toV3 is SmartAccountTestLab, ArbitrumForkSettings {
+            Nexus(payable(address(smartAccountV2))).isModuleInstalled(MODULE_TYPE_VALIDATOR, address(VALIDATOR_MODULE), ""),
             "Validator module should be installed after upgrade."
         );
     }
 
-    function test_AccountIdValidationAfterUpgrade() public {
-        test_UpgradeV2ToV3AndInitialize(); // Ensure the smartAccountV2 is upgraded and initialized
-
-        string memory expectedAccountId = "biconomy.nexus.0.0.1";
-        string memory actualAccountId = IAccountConfig(payable(address(smartAccountV2))).accountId();
-        assertEq(actualAccountId, expectedAccountId, "Account ID does not match after upgrade.");
-    }
-
-    function test_USDCTransferPostUpgrade() public {
-        test_UpgradeV2ToV3AndInitialize(); // Ensure the setup and upgrade are complete
-
-        MockToken usdc = MockToken(USDC_ADDRESS);
-        address recipient = address(0x123); // Random recipient address
-        uint256 amount = usdc.balanceOf(address(smartAccountV2)); // Full balance for transfer
-
-        // Encode the call to USDC's transfer function
-        bytes memory callData = abi.encodeWithSelector(usdc.transfer.selector, recipient, amount);
-
-        // Create execution array
-        Execution[] memory execution = new Execution[](1);
-        execution[0] = Execution(address(usdc), 0, callData);
-
-        // Pack user operation
-        PackedUserOperation[] memory userOps = preparePackedUserOperation(
-            BOB,
-            Nexus(payable(address(SMART_ACCOUNT_V2_ADDRESS))),
-            EXECTYPE_DEFAULT,
-            execution
-        );
-
-        // Execute the operation via the EntryPoint
-        ENTRYPOINT_V_0_7.handleOps(userOps, payable(OWNER_ADDRESS));
-
-        // Check the recipient received the USDC
-        assertEq(usdc.balanceOf(recipient), amount, "USDC transfer failed");
-    }
-
-    function test_NativeEthTransferPostUpgrade() public {
-        test_UpgradeV2ToV3AndInitialize(); // Ensure the setup and upgrade are complete
-
-        address recipient = address(0x123); // Random recipient address
-        uint256 amount = 1 ether; // Amount of ETH to transfer
-
-        vm.deal(address(smartAccountV2), amount + 1 ether); // Ensure the smart account has ETH to send
-
-        // Create execution array
-        Execution[] memory execution = new Execution[](1);
-        execution[0] = Execution(recipient, amount, "");
-
-        // Pack user operation
-        PackedUserOperation[] memory userOps = preparePackedUserOperation(BOB, Nexus(payable(address(smartAccountV2))), EXECTYPE_DEFAULT, execution);
-
-        // Execute the operation via the EntryPoint
-        ENTRYPOINT_V_0_7.handleOps(userOps, payable(OWNER_ADDRESS));
-
-        // Check the recipient received the ETH
-        assertEq(address(recipient).balance, amount, "ETH transfer failed");
-    }
-
-    function preparePackedUserOperation(
```

Comment:
what was wrong with this name?

Reply:
nothing, we had many different namings `prepare`, `build`, `create`, had to choose one for consistency

Reply:
I liked prepare but should be fine. 
good if we replace such things in final version to avoid conflict
can focus on fixing tests, descriptions or solhint rules and slither in these prs.